packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -1678,6 +1679,7 @@ export const DEFAULT_COMPOSITE_ICONS: {
     ResumeCall: JSX.Element;
     BackSpace: JSX.Element;
     iconBackdrop: JSX.Element;
+    sparkle: JSX.Element;
```

Comment:
✨

Reply:
hmm I'm interested, what does this sparkle icon look like? 
Can't seem to find it in fluentui icons
![image](https://user-images.githubusercontent.com/97130533/188747684-7bec8369-8d16-4516-99dc-870396917646.png)

Reply:
Use this site for searching for icons: https://iconcloud.design/browse/Fluent%20System%20Library/Fluent%20Regular

Reply:
Ah ha! Thanks James!
![image](https://user-images.githubusercontent.com/97130533/188759040-c2a2a1a8-8e9a-495b-92a8-c2d61d56f2c4.png)

Reply:
being added in followup PR: https://github.com/Azure/communication-ui-library/pull/2307 😊